examples/default.json

```diff
@@ -0,0 +1,54 @@
+{
+  "headers": {
+    "cache-control": "public, max-age=30",
+    "content-encoding": "gzip"
+  },
+  "stages": [
+    {
+      "at": 0,
+      "status": 200,
+      "latency": "100ms"
+    }
+  ],
+  "body": "<!doctype html>
```

Comment:
What's happening here is that the JSON is technically invalid as there are new lines. 
I could have put everything in a single line, but it'd have make the readability much harder at this stage. 

I've decided to ho this way as I want to implement some placeholder logic to inject snippets of content in the JSON, using a format similar to [what done for Response compression ](https://github.com/bbc/origin_simulator#response-compression).

Will work on this on a separate iteration.